### PR TITLE
server: Use `get_extra_info` to get the `ssl_object`

### PR DESCRIPTION
**Pull request by xdustinface**

### CODE REVIEW

File: chia/server/server.py [hunk 1]
```diff
@@ -310,7 +318,12 @@ async def incoming_connection(self, request):
         ws = web.WebSocketResponse(max_msg_size=max_message_size)
         await ws.prepare(request)
         close_event = asyncio.Event()
-        cert_bytes = request.transport._ssl_protocol._extra["ssl_object"].getpeercert(True)
+        ssl_object = request.get_extra_info("ssl_object")
```
Comment (emlowe) [inline chia/server/server.py:321]: I think you can directly get the peercert here with `request.get_extra_info("peercert") - https://docs.python.org/3/library/asyncio-protocol.html?highlight=peercert#asyncio.BaseTransport.get_extra_info

Comment (xdustinface) [reply chia/server/server.py:321]: Im on mobile so i cant verify but looks like then its called with `binary_form=False` and we would need to convert it?

Comment (xdustinface) [reply chia/server/server.py:321]: https://docs.python.org/3/library/ssl.html#ssl.SSLSocket.getpeercert

Comment (emlowe) [reply chia/server/server.py:321]: Ah I see, we want `binary_form=True`, but `request.get_extra_info("peercert")` will call it as the default which is `False`. Ok resolving comment

**Review by emlowe — APPROVED** [review]

**Pull request merged**